nativelib/src/main/resources/scala-native/gc/commix/CommixGC.c

```diff
@@ -114,6 +114,14 @@ size_t scalanative_GC_get_max_heapsize() {
 
 size_t scalanative_GC_get_used_heapsize() { return Heap_getMemoryUsed(&heap); }
 
+size_t scalanative_GC_stats_collection_total() {
+    return heap.gcStats.collectionTotal;
+}
+
+size_t scalanative_GC_stats_collection_duration_total() {
```

Comment:
As I recall, our standard was to make these helper methods in C style with camel_case and lowercase letters.

Reply:
I can make a follow-up PR to change `scalanative_GC_x` to `scalanative_gc_x` across all GCs.

Reply:
I see that @WojciechMazur  changed that to match Boehm and we also use Boehm env vars so that is a good exception I think.